### ISSUE TITLE
Strings in terminal output are hard to read

### ISSUE THREAD

**Issue by kwalrath** [dart-lang/site-www#1928]
To reproduce, go to https://dart.dev/tutorials/server/get-started#2-install-dart and click the **Linux** tab. The `curl` commands are blue (#00677A) on black (#212121), which is very hard to see (2.46:1 contrast ratio, according to https://webaim.org/resources/contrastchecker/). Here's a screenshot:

![image](https://user-images.githubusercontent.com/2164483/65283609-508bff80-daec-11e9-9715-77638340f01d.png)

**Comment by chalin** [comment]
Since I was dabbling in colors earlier today, I quickly looked into this. How about the following color (#00CCEE)?

 ![image](https://user-images.githubusercontent.com/4140793/65286291-0a936380-db0d-11e9-89eb-b08f55d1a051.png)

If you like it, I can add the change to #1921, or address it separately. Let me know what you prefer.

**Comment by chalin** [comment]
The #00CCEE color makes the contrast checker happy:
> ![image](https://user-images.githubusercontent.com/4140793/65286527-e1270780-db0d-11e9-8abf-2222303041da.png)

**Comment by kwalrath** [comment]
Looks good!

**Comment by chalin** [comment]
I'll also be tweaking the name class contrast:
> <img width="400" src="https://user-images.githubusercontent.com/4140793/65328056-63054800-db83-11e9-96d3-c599920c6f94.png">
